### ISSUE TITLE
Visualizing Graph

### ISSUE THREAD

**Issue by ronykrell** [greenrobot/EventBus#644]
Is there any way to visualize the Subscriber / Publisher sequences? Something that creates a graphic similar to the one on the README where someone new to the app can quickly see sequences to better understand. 

An example: A fragment publishes an event, subscriber A and B receive it, and A then publishes a different event, etc etc

**Comment by HuSiNi** [comment]
no,because [post] [register] [subscribe] methods are called on the RunTime，not at compile time。
if  [post] [register] [subscribe] methods can be execute at compile time,then it can gennerate a Visualizing Graph by annotation processing tool (APT).

**Comment by tomridder** [comment]
no,because [post] [register] [subscribe] methods are called on the RunTime，not at compile time。
if [post] [register] [subscribe] methods can be execute at compile time,then it can gennerate a Visualizing Graph by annotation processing tool (APT).

**Comment by sunreview** [comment]
if it is Visualizing Graph ,i guess greenbot team should add the annotation of @post,because the Graph can only generate in compile time.
if it is a LOG like
**A fragment publishes an event**
**ubscriber A and B receive it, **
to record all the events, then it is so easy.
